src/etc/rc.initial.toggle_sshd

```diff
@@ -31,6 +31,21 @@ require_once("filter.inc");
 $fp = fopen('php://stdin', 'r');
 
 if (isset($config['system']['ssh']['enable'])) {
+	if (!empty(getenv('SSH_CONNECTION'))) {
+		echo <<<EOL
++------------------------------------------------------------+
+|                                                            |
+| !!! WARNING !!!                                            |
+|                                                            |
+| You are currently connected via SSH! If you disable it,    |
+| you risk locking yourself out of the system. Please ensure |
+| you have another way to access the console if necessary.   |
+|                                                            |
++------------------------------------------------------------+
```

Comment:
No need to get that fancy with the ASCII art. The text should be run through `gettext()` so it can be translated and aligning it that way is not viable. A line above/below is OK and likely sets it apart enough.

Also, please reword it to avoid addressing the user directly. Perhaps:

> This session is currently established via SSH. Disabling SSH will terminate this session and prevent future SSH connections from any user. Before disabling SSH ensure that administrators have alternate means of accessing the firewall, such as the GUI or serial console.

Reply:
Good points, I actually realized this after testing it from my mobile phone which had a 40 char terminal width. I'll make changes.

Reply:
@jim-p one clarification: the rest of the console menu doesn't appear to be filtered through `gettext()`. So is that really necessary? I see only the changelog that gets saved to config.xml gets interpreted.

Reply:
> @jim-p one clarification: the rest of the console menu doesn't appear to be filtered through `gettext()`. So is that really necessary? I see only the changelog that gets saved to config.xml gets interpreted.

It just hasn't been converted. It's best to add it for new things as we go rather than wait and make the eventual conversion an even larger job. Also it's better if a scary warning gets translated so it's more clearly understood by the user.